src/modules/functions.js

```diff
@@ -0,0 +1,84 @@
+// Getting data from the theMealDB API function
+export const getMealData = async () => {
+  const request = new Request('https://themealdb.com/api/json/v1/1/search.php?f=c');
+  const response = await fetch(request);
+  const data = await response.json();
+  return data.meals;
+};
+
+// Populating items in the DOM function
+export const populatedishes = async (mealData) => {
+  const foodListSection = document.querySelector('.food-list');
+  mealData.forEach((element) => {
+    const foodCard = document.createElement('div');
+    foodCard.classList.add('food-card', 'flex-column');
+    foodCard.innerHTML = `<div class = 'image-box'><img src = '${element.strMealThumb}' alt = 'meal'></div>
+                              <div class = 'flex-row meal-title-and-interactions-box'>
+                                <h2 class = 'meal-name'>${element.strMeal}</h2>
+                                <i id ="M${element.idMeal}" class="fa-sharp fa-solid fa-heart heart" style="color: #8b4513;"></i><span id ="L${element.idMeal}" class = "likes">0 likes</span>
+                              </div>
+                              <div class = "buttons-box flex-row">
+                                <button id = "B${element.idMeal}"  class = 'comment'>Comments</button>
+                                <button class = 'Reservation'>Reservation</button>
+                              </div>
```

Comment:
- [ ] [OPTIONAL] It will be better if you can remove the `Reservation` button as it is not implemented.
![image](https://github.com/nilab-khwaja/Js-Capstone/assets/43995830/c45a10df-5079-4836-a017-5b232b258784)